vllm/worker/model_runner.py

```diff
@@ -547,17 +675,21 @@ def build(self) -> ModelInputForGPU:
             batch_size = graph_batch_size
 
         # Tokens and positions.
-        input_tokens.extend([0] * cuda_graph_pad_size)
-        input_positions.extend([0] * cuda_graph_pad_size)
+        if cuda_graph_pad_size:
+            input_tokens.extend([0] * cuda_graph_pad_size)
+            input_positions.extend([0] * cuda_graph_pad_size)
```

Comment:
Can avoid allocating lists here, same below
```suggestion
            input_tokens.extend(itertools.repeat(0, cuda_graph_pad_size))
            input_positions.extend(itertools.repeat(0, cuda_graph_pad_size))
```

Reply:
Nice trick, did not knew about this one.